lib/build/source-inclusion.js

```diff
@@ -27,8 +28,16 @@ exports.SourceInclusion = class {
     this.items.push(item);
   }
 
+  isExcluded(item){
+	  let found = this.excludes.findIndex(x => {			
+			let exclusion = new this.bundle.Minimatch(x);
```

Comment:
Part of it might be my lack of knowledge of Minimatch, but I'm not entirely sure how the best way to go about creating the matcher up front. Where the `include` matcher can have a single pattern and loop through an array of files, the `exclude` matcher needs to have a file loop through an array of `exclusions`.  
Perhaps it's the Friday in me. I think I might be able to create the exclusion list on the bundle itself (instead of the source-inclusion), then each exclusion is just created once, and when the source-inclusion goes through trying to match files, it'll match against the bundle's exclusion list. How does that approach sound @EisenbergEffect?